Dockerfile-ngc-hpc

```diff
@@ -3,6 +3,7 @@ FROM ${BASE_IMAGE}
 
 # Copy various shell scripts that group dependencies for install
 COPY dockerfile_scripts /tmp/det_dockerfile_scripts
+RUN pip install -r /tmp/det_dockerfile_scripts/requirements.txt
```

Comment:
This doesn't have to be here either, as these images are always built from either `Dockerfile-pytorch-ngc` or `Dockerfile-tensorflow-ngc`